src/main/scala/com/typesafe/sbt/web/SbtWeb.scala

```diff
@@ -318,16 +298,34 @@ object SbtWeb extends AutoPlugin {
     nodeModules := nodeModuleGenerators(_.join).map(_.flatten).value
   )
 
+  def webJarsPathPrefix: Def.Initialize[Task[String]] = Def.task {
+    path(s"$WEBJARS_PATH_PREFIX/${moduleName.value}/${version.value}/")
+  }
+
+  def syncExportedAssets(): Def.Initialize[Task[File]] = Def.task {
+    val syncTargetDir = classDirectory.value
+    val syncRequired = TrackLevel.intersection(trackInternalDependencies.value, exportToInternal.value) match {
```

Comment:
This doesn't look right. Let's say `project A` depends on `project B` and `project C`.
`trackInternalDependencies` key sets the track level of `project A` tracking `project B` and `C`. So if `A` doesn't want to trigger cascading compilation, it can just say `trackingInternalDepndencies := TrackLevel. NoTracking`.

On the other hand, `exportToInternal.value` sets the tracking level from the dependee's point of view. Suppose `project B` is takes really really long time, and it doesn't want any projects to cause cascading compilation. It can opt-out of it by setting `exportToInternal := TrackLevel.NoTracking`. See https://github.com/sbt/sbt/pull/2354/commits/98b26ae5d3009cbfd5a062cae108871d3ad3e466#diff-6373e7f7122325e753b75fe1cc76ff5fR1547